utils.py

```diff
@@ -131,5 +133,13 @@ def match_field_dl_vlan(value, field_flow):
     0 is not allowed for value. """
     if not value:
         return field_flow == 0
+    if isinstance(value, list):
```

Comment:
In the last commit, this condition is removed since `dl_vlan` is always forced to be a [list.](https://github.com/kytos-ng/sdntrace_cp/blob/684db538fb49e264187c18c291e2e86a478ee904/utils.py#L46)